app/assets/javascripts/discourse/tests/acceptance/user-preferences-tracking-test.js

```diff
@@ -196,6 +196,22 @@ acceptance("User Preferences - Tracking", function (needs) {
     });
   });
 
+  test("additional precedence option when one category is watched and tag is muted", async function (assert) {
+    this.siteSettings.tagging_enabled = true;
+
+    await visit("/u/eviltrout/preferences/tracking");
+
+    const mutedTagsSelector = selectKit(
+      ".tracking-controls__muted-tags .tag-chooser"
+    );
+
+    assert.notOk(exists(".user-preferences__watched-precedence-over-muted"));
```

Comment:
```suggestion
    assert.dom(".user-preferences__watched-precedence-over-muted").doesNotExist;
```

Small one but qunit-dom is so much nicer to use and read :)